reactor-core/src/main/java/reactor/core/publisher/DelegateSinkFluxProcessor.java

```diff
@@ -47,17 +47,17 @@ public Context currentContext() {
 
 	@Override
 	public void onComplete() {
-		sink.emitComplete();
+		sink.tryEmitComplete();
 	}
 
 	@Override
 	public void onError(Throwable t) {
-		sink.emitError(t);
+		sink.tryEmitError(t);
 	}
 
 	@Override
 	public void onNext(IN in) {
-		sink.emitNext(in);
+		sink.tryEmitNext(in);
```

Comment:
IMO this and many other usages should use the regular emitNext that would fail on overflow. Otherwise we're introducing the same problem as in M2

Reply:
good catch, indeed and same for `DelegateSinkOneMonoProcessor`

Reply:
this will actually greatly simplify the PR since most uses in tests will be back to calling `emitXxx`

Reply:
I've rebased so that a single commit has the introduction of tryEmit BUT correct use of emit in test classes